js/templates/modals/baseModal.html

```diff
@@ -1,4 +1,22 @@
+<div class="popInMessageHolder js-popInMessages"></div>
 <div class="<%= ob.modelContentClass %>">
-  <i class="jsModalClose <%= ob.closeButtonClass %> <% !ob.showCloseButton && print('hide') %>"></i>
+  <div class="topControls pureFlex spread js-closeClickTarget js-btcTicker">
```

Comment:
Yeah, this IMHO absolutely should not go into the baseModal. The baseModal is intended to be very generic and stuff should only go in there that would be applicable for every modal. Certain modals will want to add their own custom content and that should be done in their respective templates.

Doing it this way would would mean the base modal would just keep growing to handle every different permutation we'd need. The general object-oriented approach is to centralize any common functionality into a base class and then have child classes specify the differences.

Could you please revert this to how it was before as well as moving the respective parts that were brought in here back to where they were? Once those are back in place, they should serve as a good example of how to implement some custom topControls content into the about modal. If there's any specific stumbling block you are running into, please let us know and we'll be happy to help.